script/load_templates

```diff
@@ -141,6 +141,25 @@ sub post_entry{
 
     my %param;
 
+    if ($table eq 'JobTemplates') {
+        unless (defined($entry->{prio})) {
+            # we have to migrate the prio from the TestSuite to the JobTemplate
+            for my $ts (@{$info->{TestSuites}}) {
+                if ($ts->{name} eq $entry->{test_suite}{name}) {
+                    $entry->{prio} = $ts->{prio};
+                }
+            }
+        }
+        unless (defined($entry->{group_name})) {
```

Comment:
Yeah, technically DBIx takes only undef as null, so defined check is enough. So is '' supposed to be valid group name?